env0/data_template.go

```diff
@@ -83,6 +83,12 @@ func dataTemplate() *schema.Resource {
 				Description: "if specified, will only retry (on destroy) if error matches specified regex",
 				Computed:    true,
 			},
+			"github_installation_id": {
+				Type:        schema.TypeInt,
+				Description: "The installation id env0 application on the repository",
+				Computed:    true,
```

Comment:
Why [`Computed`](https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#computed)?

Reply:
b8eb7e24e86f888217bb444ede4084e39046e515 you right